CONTRIBUTING.md

```diff
@@ -43,6 +42,10 @@ Once you have built your template file or files, the commit message should have
 
 * for template modifications without a version change, use ```<pkgname>: <reason>``` ([example](https://github.com/void-linux/void-packages/commit/8b68d6bf1eb997cd5e7c095acd040e2c5379c91d)).
 
+* for package removals, use ```<pkgname>: remove package``` ([example](https://github.com/void-linux/void-packages/commit/cbf7353bad4c9297af8b1b395b4d87504e812358)).
```

Comment:
This is better example that does not turn package into empty: https://github.com/void-linux/void-packages/commit/83784632d94deee5d038c8e1c4c1dffa922fca21

Reply:
Ok, thanks!